docs/DeveloperGuide.adoc

```diff
@@ -90,7 +111,9 @@ The `UI` component,
 
 [[Design-Logic]]
 === Logic component
-
+This section explains the design of the Logic component. +
+Figure 5 below depicts the classes in the Logic component and their relationships to each other.
+It will be referred to in our explanation of the Logic component design.
```

Comment:
same here , think line 116 is not needed.
line 115: "their relationship with one another."

Reply:
Changed